architecture/observability/access-control-to-metrics/README.md

```diff
@@ -0,0 +1,187 @@
+# Access control to metrics in NERC
+
+## Use case
+
+Protect and expose the ACM Observability metrics data through a secure proxy providing fine-grained resource permissions to the metrics.
+
+## Background
+
+The New England Research Cloud is a perfect environment for software cloud related research.
+Because the metrics data for the New England Research Cloud OpenShift clusters is only found in the infra cluster behind a VPN,
```

Comment:
```suggestion
Because the metrics data for the New England Research Cloud OpenShift clusters is only found in the infra cluster behind a VPN, we wish to protect and expose the metrics data through a secure proxy providing fine-grained resource permissions to the metrics.
```